test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1

```diff
@@ -1,4 +1,24 @@
-﻿Describe "Implicit remoting and CIM cmdlets with AllSigned and Restricted policy" -tags "Feature" {
+﻿#
+# Ensure WinRM remoting is enabled
+#
+if ($IsWindows)
```

Comment:
You should have this in `BeforeAll`

Reply:
I believe BeforeAll blocks must be inside a Describe block.  But this file has multiple Describe blocks and I want to enable PSRemoting only once.  This is the only way I know how to do this, but let me know if there is an alternative method.

Reply:
I see, perhaps then, it might be better to create a helper function called `Enable-PSCoreRemoting` that wraps this and have each BeforeAll call that?

Reply:
It is not necessary to do this check for each remoting test ... the remoting tests are slow enough as they are.  This check should really only be done once per test run.  Do we have any set up code before we start CI or Feature tests?  That might be a better place to ensure remoting is enabled instead of having this check in each remoting test file.

Reply:
The check itself shouldn't be that slow, right?  If the endpoint is created, it won't create it again.

Reply:
Get-PSSessionConfiguration is slow and in any case should not need to be done before each remoting test.  If the state of remoting changes during a test run then we need to fix the test that altered the state.  Note that this is the same pattern we use in other tests (see PSSessionConfiguration.Tests.ps1)

Reply:
I'm ok with this.  Thanks.